plugin-packs/postcss-preset-env/src/lib/get-options-for-browsers-by-feature.mjs

```diff
@@ -3,7 +3,8 @@ import { getUnsupportedBrowsersByFeature } from './get-unsupported-browsers-by-f
 
 // add extra options for certain browsers by feature
 export default function getOptionsForBrowsersByFeature(browsers, feature, cssdbList, options, logger) {
-	const supportedBrowsers = browserslist(browsers, { ignoreUnknownVersions: true });
+	const env = browsers ? undefined : options.env; // When `browsers` is set, ignore `env`
```

Comment:
Instead of this, I could have added an `env` parameter to `getOptionsForBrowsersByFeature` and it's calling function `formatStagedFeature`.

That would match how we pass browsers through the functions but felt slightly clunky - these functions have enough parameters already so I didn't feel great about adding another one given that the env is already present in the options.

This logic matches what is in https://github.com/csstools/postcss-plugins/blob/cd3f9fcf134a485c8f0240ccef9f7974c21d5e7b/plugin-packs/postcss-preset-env/src/lib/list-features.mjs#L17